src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -1947,12 +1947,12 @@ static void dump_global_settings(void)
     fprintf(stdout, "max_buffered_send: %" PRIu64 "\n", MPIDI_OFI_global.max_buffered_write);
     fprintf(stdout, "max_buffered_write: %" PRIu64 "\n", MPIDI_OFI_global.max_buffered_send);
     fprintf(stdout, "max_msg_size: %" PRIu64 "\n", MPIDI_OFI_global.max_msg_size);
-    fprintf(stdout, "max_order_raw: %zd\n", MPIDI_OFI_global.max_order_raw);
-    fprintf(stdout, "max_order_war: %zd\n", MPIDI_OFI_global.max_order_war);
```

Comment:
They can't have value `-1` because they are typed as `size_t` : https://github.com/pmodels/mpich/blob/d7955c8f9ac5d838286448b2a7e8fa8e3d0267f7/src/mpid/ch4/netmod/ofi/ofi_types.h#L319-L321

How about let's change the type to `MPI_Aint`?

Reply:
Yeah, that is weird. https://ofiwg.github.io/libfabric/v1.12.1/man/fi_endpoint.3.html also uses `size_t`, but have clearly listed the -1 special value under **Max RMA Ordered Size** section.

Agreed to change to MPI_Aint.

Reply:
I guess we can delete the first commit, since all `size_t` are changed to `MPI_Aint` in the second commit?